meta/store.go

```diff
@@ -332,7 +332,7 @@ func (s *Store) joinCluster() error {
 
 	// We already have a node ID so were already part of a cluster,
 	// don't join again so we can use our existing state.
-	if s.id != 0 || raft.PeerContained(s.peers, s.RemoteAddr.String()) {
```

Comment:
Because if our address was in the list of peers, we would not join or initialize.